### PR TITLE
Add new interaction tokens + update skin schema

### LINKED ISSUE #1202: Cards [Hover and pressed states]
## Description

Add hover state in cards components without buttons
- Media card
- Data card
- Display data card
- Display media card
- Snap card
- Poster card
- Highlighted card

### Highlighted, SnapCard and displayMediaCard

| Element | State | Default ∨ OverInverse | inverse | Inverse/OverInverse |  OverAlternative |
|--------|--------|--------|--------|--------|------- |
| background | `hover` | `backgroundContainerHover` | `backgroundContainerBrandHover` | `backgroundContainerBrandOverInverseHover` | `backgroundContainerAlternativeHover` |
| background | `:active`/`pressed` | `backgroundContainerPressed` | `backgroundContainerBrandPressed` | `backgroundContainerBrandOverInversePressed` | `backgroundContainerAlternativePressed` |

### MediaCard and DataCard

| Element | State | Default ∨ OverInverse | OverAlternative |
|--------|--------|--------| ------ |
| background | `hover` | `backgroundContainerHover`|  `backgroundContainerAlternativeHover`|
| background | `:active`/`pressed` | `backgroundContainerPressed` | `backgroundContainerAlternativePressed`|

### Changes in displayMediaCard and posterCard

Add overlay with the following values

| State | Value (hex) | Value (with applyAlpha) |
| ----- | ---- | ---- |
| `:hover` | `#00000040` | `applyAlpha(#000000, 0.25)` |
| `:active`/`pressed` | `#00000059` | `applyAlpha(#000000, 0.35)` |

**Overlay values doesn't change between dark and light modes**

## Transitions

* All `:hover` and `:active` / `pressed` states have an `ease-in-out` transition timing function.
* `:hover` states have a transition duration of 150ms
* `:active`/`pressed` states have a transition duration of 100ms

## Palette values for all brands

tokens-states

## Tasks

- [x] Update specs
- [x] [Android ticket](https://jira.tid.es/browse/ANDROID-12685)
- [x] [iOS ticket](https://jira.tid.es/browse/IOS-8746)
- [x] [Web ticket](https://jira.tid.es/browse/WEB-831)

## Fixes

- [x] https://github.com/Telefonica/mistica-design/issues/1215

### LINKED ISSUE #1204: Lists [Hover and pressed states]
### Tasks

**Update/14/06/23**

**Due to changes in the cards states definition this docs should be updated**

https://www.figma.com/file/Be8QB9onmHunKCCAkIBAVr/branch/LE1mzniQNIUKpmKqqVRIrd/%F0%9F%94%B8-Lists-Specs?type=design&node-id=0%3A1140&t=ZMe2llVee3Nb3UmT-1

## Tasks

- [x] Update specs
- [x] [Android ticket](https://jira.tid.es/browse/ANDROID-13235)
- [x] [iOS ticket](https://jira.tid.es/browse/IOS-8940)
- [x] [Web ticket](https://jira.tid.es/browse/WEB-1390)

## Related tasks

- [x] https://github.com/Telefonica/mistica-design/issues/1072

### PR DESCRIPTION
**Pull request by aweell**
tokens-states

Solves:
* https://github.com/Telefonica/mistica-design/issues/1202
* https://github.com/Telefonica/mistica-design/issues/1204

### CODE REVIEW

File: tokens/blau.json [hunk 1]
```diff
@@ -25,11 +25,41 @@
...
+      "value": "{palette.blauBluePrimary70}",
+      "type": "color",
+      "description": "blauBluePrimary70"
+    },
+    "backgroundContainerBrandHoverOverInverse": {
+      "value": "{palette.blauBlueSecondary50}",
+      "type": "color",
+      "description": "blauBlueSecondary50"
+    },
+    "backgroundContainerBrandPressedOverInverse": {
```
Comment (yceballost) [inline tokens/blau.json]: Should the state be at the end of the token name? 🤔

Comment (aweell) [reply tokens/blau.json]: Yes, I updated following the semantics of other tokens with state + themevariant

**Review by yceballost — APPROVED** [review]

File: tokens/o2.json [hunk 2]
```diff
@@ -517,16 +557,56 @@
       "type": "color",
       "description": "darkModeGrey"
     },
+    "backgroundContainerHover": {
+      "value": "{palette.darkModeGrey7}",
+      "type": "color",
+      "description": "darkModeGrey"
```
Comment (AnaMontes11) [inline tokens/o2.json]: Change description to "darkModeGrey7"

File: tokens/o2.json [hunk 3]
```diff
@@ -517,16 +557,56 @@
...
+    },
+    "backgroundContainerPressed": {
+      "value": "{palette.darkModeGrey8}",
+      "type": "color",
+      "description": "darkModeGrey"
```
Comment (AnaMontes11) [inline tokens/o2.json]: change description to "darkModeGrey8"

**Review by AnaMontes11 — APPROVED** [review]
Really nice!

File: tokens/telefonica.json [hunk 4]
```diff
@@ -795,7 +875,7 @@
     "feedbackInfoBackground": {
       "value": "{palette.grey8}",
       "type": "color",
-      "description": "grey8"
+      "description": "darkModeGrey8"
```
Comment (yceballost) [inline tokens/telefonica.json]: seems wrong

**Review by yceballost — CHANGES_REQUESTED** [review]

**Review by yceballost — APPROVED** [review]
letsss goooo!

**Pull request merged**